ci/release-image/entrypoint.sh

```diff
@@ -18,4 +18,13 @@ if [ "${DOCKER_USER-}" ]; then
   fi
 fi
 
+# Allow users to have scripts run on container startup to prepare workspace.
+# https://github.com/coder/code-server/issues/5177
+chmod u+x ${ENTRYPOINTD}/*.sh
+sudo chown -R ${USER} ${ENTRYPOINTD}/*
```

Comment:
Do you think we could omit these?  I think it may be reasonable to let the user assume responsibility for having the correct bits on their scripts (but let me know if there is an edge case I am missing).

Reply:
Probably not, plus sourcing the script with the `bash` prefix does not require the `x` bit to be set.